{"name": "OrangeHRM Login", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))\n", "trace": "  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\StepDefn_Orangehrm.py\", line 26, in the_user_verifies_logo\n    logo = context.driver.find_element(By.XPATH,'//div[@id=\"divLogo\"]/img')\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 857, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 433, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 344, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 366, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\request.py\", line 78, in request\n    return self.request_encode_body(\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\request.py\", line 170, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\poolmanager.py\", line 376, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\connectionpool.py\", line 787, in urlopen\n    retries = retries.increment(\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\util\\retry.py\", line 550, in increment\n    raise six.reraise(type(error), error, _stacktrace)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\packages\\six.py\", line 769, in reraise\n    raise value.with_traceback(tb)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\connectionpool.py\", line 703, in urlopen\n    httplib_response = self._make_request(\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\connectionpool.py\", line 449, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\connectionpool.py\", line 444, in _make_request\n    httplib_response = conn.getresponse()\n  File \"C:\\Users\\Vigne\\AppData\\Local\\Programs\\Python\\Python39\\lib\\http\\client.py\", line 1347, in getresponse\n    response.begin()\n  File \"C:\\Users\\Vigne\\AppData\\Local\\Programs\\Python\\Python39\\lib\\http\\client.py\", line 307, in begin\n    version, status, reason = self._read_status()\n  File \"C:\\Users\\Vigne\\AppData\\Local\\Programs\\Python\\Python39\\lib\\http\\client.py\", line 268, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"C:\\Users\\Vigne\\AppData\\Local\\Programs\\Python\\Python39\\lib\\socket.py\", line 704, in readinto\n    return self._sock.recv_into(b)\n"}, "steps": [{"name": "Given the user launches Application", "status": "passed", "start": 1659800489265, "stop": 1659800545349}, {"name": "Then the user verifies logo", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))\n", "trace": "  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\StepDefn_Orangehrm.py\", line 26, in the_user_verifies_logo\n    logo = context.driver.find_element(By.XPATH,'//div[@id=\"divLogo\"]/img')\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 857, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 433, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 344, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 366, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\request.py\", line 78, in request\n    return self.request_encode_body(\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\request.py\", line 170, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\poolmanager.py\", line 376, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\connectionpool.py\", line 787, in urlopen\n    retries = retries.increment(\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\util\\retry.py\", line 550, in increment\n    raise six.reraise(type(error), error, _stacktrace)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\packages\\six.py\", line 769, in reraise\n    raise value.with_traceback(tb)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\connectionpool.py\", line 703, in urlopen\n    httplib_response = self._make_request(\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\connectionpool.py\", line 449, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"C:\\Users\\Vigne\\PycharmProjects\\BehaveBDD\\venv\\lib\\site-packages\\urllib3\\connectionpool.py\", line 444, in _make_request\n    httplib_response = conn.getresponse()\n  File \"C:\\Users\\Vigne\\AppData\\Local\\Programs\\Python\\Python39\\lib\\http\\client.py\", line 1347, in getresponse\n    response.begin()\n  File \"C:\\Users\\Vigne\\AppData\\Local\\Programs\\Python\\Python39\\lib\\http\\client.py\", line 307, in begin\n    version, status, reason = self._read_status()\n  File \"C:\\Users\\Vigne\\AppData\\Local\\Programs\\Python\\Python39\\lib\\http\\client.py\", line 268, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"C:\\Users\\Vigne\\AppData\\Local\\Programs\\Python\\Python39\\lib\\socket.py\", line 704, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1659800545349, "stop": 1659800551409}, {"name": "When the user enters username in username field", "status": "skipped", "start": 1659800551425, "stop": 1659800551425}, {"name": "Then the user enters password in password field", "status": "skipped", "start": 1659800551425, "stop": 1659800551425}, {"name": "And the user clicks on login button", "status": "skipped", "start": 1659800551425, "stop": 1659800551425}], "start": 1659800489264, "stop": 1659800551425, "uuid": "fe3a64de-42b2-4e94-b68c-ae9e08ed087e", "historyId": "d508db5ea7f39ccfff91812f01a1919c", "fullName": "OrangeHRM Testcases: OrangeHRM Login", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "OrangeHRM Testcases"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}